src/TestCases/Traits/AssertTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace Eonx\TestUtils\TestCases\Traits;
+
+use Eonx\TestUtils\Constraints\ArraySameWithDates;
+use Eonx\TestUtils\TestCases\Exceptions\InvalidParentClassException;
+use Eonx\TestUtils\TestCases\TestCase;
+
+trait AssertTrait
+{
+    /**
+     * AssertTrait constructor.
+     */
+    public function __construct()
+    {
+        if ($this instanceof TestCase === false) {
+            throw new InvalidParentClassException('AssertTrait must be used by an Eonx\\TestUtils\\TestCases class.');
+        }
+    }
+
+    /**
+     * Assert that two arrays provided are same with flat dates.
+     *
+     * @param mixed[] $expected
+     * @param mixed[] $actual
+     *
+     * @return void
+     *
+     * @throws \Exception
+     */
+    public function assertArraySameWithDates(array $expected, array $actual): void
```

Comment:
Normally I'd be in favour of these being instance methods, but in keeping with the phpunit standards, they should be static.

Reply:
Agree. We should follow the same pattern of the tool we're working with here.